tests/drivers/spi/spi_loopback/testcase.yaml

```diff
@@ -13,6 +13,9 @@ tests:
   drivers.spi.loopback: {}
   drivers.spi.loopback.internal:
     filter: CONFIG_SPI_LOOPBACK_MODE_LOOP
+  drivers.spi.loopback.lpspi:
+    filter: DT_HAS_NXP_LPSPI_ENABLED
+    platform_allow: imx95_evk/mimx9596/m7
```

Comment:
WHy is this change needed?

Reply:
You are right. These are not needed. Will remove.
Thanks.